auth-api/pom.xml

```diff
@@ -151,9 +143,9 @@
             <Bundle-Activator>io.stargate.auth.api.AuthApiActivator</Bundle-Activator>
             <Embed-Transitive>true</Embed-Transitive>
             <Import-Package><![CDATA[
-                            !org.slf4j,
-                            !org.slf4j.helpers,
-                            !org.slf4j.spi,
+                            org.slf4j,
+                            org.slf4j.helpers,
+                            org.slf4j.spi,
                             !ch.qos.logback.classic.*,
                             !ch.qos.logback.core.*,
```

Comment:
This can be removed now as well.. As `logback-classic` is not anymore on the classpath this becomes obsolete..